test_scripts/Polices/External_UCS/003_ATF_P_TC_LPT_Creation_PreloadedPT_without_disallowed_by_external_consent_entities_off_struct.lua

```diff
@@ -0,0 +1,137 @@
+---------------------------------------------------------------------------------------------
+-- Requirement summary:
+-- [Policies] External UCS: PreloadedPT without "disallowed_by_external_consent_entities_off" struct
+--
+-- Description:
+-- In case:
+-- SDL uploads PreloadedPolicyTable without "disallowed_by_external_consent_entities_off:
+-- [entityType: <Integer>, entityId: <Integer>]" -> of "<functional grouping>" -> from "functional_groupings" section
+-- SDL must:
+-- a. consider this PreloadedPT as valid (with the pre-conditions of all other valid PreloadedPT content)
+-- b. do not create this "disallowed_by_external_consent_entities_off: [entityType: <Integer>, entityId: <Integer>]"
+-- field of the corresponding "<functional grouping>" in the Policies database.
+--
+-- Preconditions:
+-- 1. Stop SDL
+-- 2. Modify PreloadedPolicyTable (remove 'disallowed_by_external_consent_entities_off' section)
+--
+-- Steps:
+-- 1. Start SDL
+-- 2. Check SDL status
+-- 3. Register app
+-- 4. Activate app
+-- 5. Verify PTSnapshot
+--
+-- Expected result:
+-- a. Status = 1 (SDL is running)
+-- b. PTSnapshot doesn't contain 'disallowed_by_external_consent_entities_off' section
+---------------------------------------------------------------------------------------------
+
+--[[ General configuration parameters ]]
+  config.deviceMAC = "12ca17b49af2289436f303e0166030a21e525d266e209267433801a8fd4071a0"
+  config.defaultProtocolVersion = 2
+
+--[[ Required Shared Libraries ]]
+  local commonFunctions = require('user_modules/shared_testcases/commonFunctions')
+  local commonSteps = require('user_modules/shared_testcases/commonSteps')
+  local commonPreconditions = require('user_modules/shared_testcases/commonPreconditions')
+  local sdl = require('SDL')
+  local testCasesForExternalUCS = require('user_modules/shared_testcases/testCasesForExternalUCS')
+
+--[[ Local variables ]]
+  local grpId = "Location-1"
+  local checkedSection = "disallowed_by_external_consent_entities_off"
+
+--[[ General Precondition before ATF start ]]
+  commonFunctions:SDLForceStop()
+  commonSteps:DeleteLogsFileAndPolicyTable()
+  commonPreconditions:BackupFile("sdl_preloaded_pt.json")
+  testCasesForExternalUCS.removePTS()
+
+--[[ General Settings for configuration ]]
+  Test = require("user_modules/connecttest_resumption")
+  require('user_modules/AppTypes')
+
+--[[ Preconditions ]]
+  commonFunctions:newTestCasesGroup("Preconditions")
+
+  function Test:CheckSDLStatus()
+    testCasesForExternalUCS.checkSDLStatus(self, sdl.RUNNING)
+  end
+
+  function Test:StopSDL()
+    testCasesForExternalUCS.ignitionOff(self)
+  end
+
+  function Test:CheckSDLStatus()
+    testCasesForExternalUCS.checkSDLStatus(self, sdl.STOPPED)
+  end
+
+  function Test.RemoveLPT()
```

Comment:
@istoimenova, Not accepted, Snapshot is removed by usage of testCasesForExternalUCS.removePTS() function